javav2/usecases/Creating_serverless_aurora_app/Readme.md

```diff
@@ -53,6 +62,9 @@ To use the **RdsDataClient** object, you require these two ARN values:
 
 **Note:** You must set up inbound rules for the security group to connect to the database. You can set up an inbound rule for your development environment. Setting up an inbound rule essentially means enabling an IP address to use the database. Once you set up the inbound rules, you can connect to the database from a client. For information about setting up security group inbound rules, see [Controlling Access with Security Groups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.RDSSecurityGroups.html).  
```

Comment:
"Once you set up the inbound rules," --> "After you set up the inbound rules," 

Sentence case for this link title: 
[Controlling access with security groups]

Reply:
fixed